packages/graphiql/test/index.html

```diff
@@ -105,15 +109,16 @@
       }
 
       // Defines a GraphQL fetcher using the fetch API.
+      const graphQLEndpoint = window.location.protocol + '//' + window.location.host + '/graphql';
       function graphQLFetcher(graphQLParams) {
-        return fetch('/graphql', {
+        return fetch(graphQLEndpoint, {
           method: 'post',
           headers: {
             'Accept': 'application/json',
             'Content-Type': 'application/json',
           },
           body: JSON.stringify(graphQLParams),
-          credentials: 'include',
+          credentials: 'same-origin',
```

Comment:
If we need to test against a different server (e.g. one that supports subscriptions :wink:) then this is necessary to allow that. Should not change behaviour for the current situation.